addons/sale/wizard/sale_quotation_send.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+class SaleQuotationSend(models.TransientModel):
+    _name = 'sale.quotation.send'
+    _description = 'Sale Quotation Send'
+
+    active_draft_ids = fields.Many2many(
+        string="Selected quotations draft",
+        comodel_name='sale.order',
+        relation='sale_order_draft_sale_quotation_send_rel',
+        help="List of the selected quotation's ids in the state \"draft\".",
+    )
+    active_sale_ids = fields.Many2many(
+        string="Sale Orders",
+        comodel_name='sale.order',
+        relation='sale_order_sale_sale_quotation_send_rel',
+        help="List of the selected quotation's ids in the state \"sent\" or \"sale\".",
+    )
+    draft_template_id = fields.Many2one(
+        string="Use template for Quotations",
+        comodel_name='mail.template',
+        index=True,
+        domain='[("model", "=", "sale.order")]'
+    )
+    sale_template_id = fields.Many2one(
+        string="Use template for Sale Orders",
+        comodel_name='mail.template',
+        index=True,
+        domain='[("model", "=", "sale.order")]'
+    )
+
+    @api.model
+    def default_get(self, fields):
+        default = super(SaleQuotationSend, self).default_get(fields)
+        active_draft_ids = self._context.get('active_draft_ids')
+        active_sale_ids = self._context.get('active_sale_ids')
+
+        draft_template_id = self._context.get('default_draft_template_id')
+        sale_template_id = self._context.get('default_sale_template_id')
+
+        if not active_draft_ids and not active_sale_ids:
+            raise UserError(_("You should select at least one quotation."))
+
+        default.update({
+            'active_draft_ids': active_draft_ids,
+            'active_sale_ids': active_sale_ids,
+            'draft_template_id': draft_template_id,
+            'sale_template_id': sale_template_id,
+        })
+
+        return default
+
+    def send_quotation_action(self):
+        self.ensure_one()
+
+        if self.active_draft_ids:
+            for quotation in self.active_draft_ids:
+                responsible_emails_set = {user.email for user in filter(None, \
+                    (quotation.user_id, self.env.ref('base.user_admin', raise_if_not_found=False)))}
```

Comment:
Too many instructions at once may confuse the reader (that was my case).
I get that you're filtering out missing emails from the set, but a different approach can get you the same result with better clarity. For example:
```suggestion
                admin_user = self.env.ref('base.user_admin', raise_if_not_found=False)
                responsible_emails_set = {user.email for user in (quotation.user_id, admin_user) if user is not None}
```